docs/index.md

```diff
@@ -132,6 +137,34 @@ Instead of using babel-node in production, we recommend using Babel to transpile
 </aside>
 
 
+## Offline functionality
+
+RippleAPI can also function without internet connectivity. This can be useful in order to generate secrets and sign transactions from a secure, isolated machine.
+
+To instantiate RippleAPI in offline mode, use the following boilerplate code:
+
+```javascript
+const {RippleAPI} = require('ripple-lib');
+
+const api = new RippleAPI();
+/* insert code here */
+```
+
+Methods that depend on the state of the Ripple Consensus Ledger are unavailable in offline mode. To prepare transactions offline, you **must** specify  the `fee` and `sequence` in the [transaction instructions](#transaction-instructions). The following methods should work offline:
```

Comment:
It also requires `maxLedgerVersion` because it queries `getLedgerVersion` and adds 3 by default.